test-helpers/a11y-helper.js

```diff
@@ -0,0 +1,40 @@
+/*eslint no-console: 0*/
+/*eslint no-undef: 0*/
+import 'axe-core/axe.min.js';
+
+/*
+* Rules can be disabled using the options parameter:
+* options = { "rules":
+*   {
+*     "color-contrast": { enabled: false },
+*     "valid-lang": { enabled: false }
+*   }
+* }
+* Rule ids can be found here: https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md
+*/
+export function runAxe(element, callback, options) {
```

Comment:
You might be able to use `await` / `async` here:

    const results = await axe.run( ... );